### PR TITLE
UpdatedComponent: translation and lookbook preview fix

### PR DESCRIPTION
**Pull request by deepsidhu85**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR updates the translation for `UpdatedComponent` to use scope for distance_in_words instead of having 2 translations joined. Fixed lookbook preview for `UpdatedComponent` and `TimeAgoComponent` so they display correctly in dark mode. Also, the custom description was removed for the `UpdatedComponent`

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

image

image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1) Ensure the translations display correctly in the lookbook previews
2) Ensure the translations display correctly for updated at in the members tables for groups and projects when the member is updated

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -203,6 +202,44 @@ en:
         direct: Direct shared
   datetime:
     distance_in_words:
```
Comment (joshsadam) [inline config/locales/en.yml:204]: Isn't this handled in the stuff Chris added in his PR: https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en-CA.yml

Comment (deepsidhu85) [reply config/locales/en.yml:204]: Yup for the default scope they are handled. The UpdatedComponent was using 2 partial translations and joining them together and well they didn't play nicely together. So I added in an updated scope under the distance_in_words section for text to be displayed when a record is updated.

Comment (joshsadam) [reply config/locales/en.yml:204]: Sounds good, thanks

**Review by joshsadam — APPROVED** [review]
This looks good to me, thanks fo updating it.

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**